build-system/sauce_connect/start_sauce_connect.sh

```diff
@@ -22,12 +22,13 @@ YELLOW() { echo -e "\033[1;33m$1\033[0m"; }
 GREEN() { echo -e "\033[0;32m$1\033[0m"; }
 RED() { echo -e "\033[0;31m$1\033[0m"; }
 
-SC_VERSION="sc-4.5.1-linux"
+SC_VERSION="sc-4.5.3-linux"
 AUTHENTICATED_STATUS_URL="https://$SAUCE_USERNAME:$SAUCE_ACCESS_KEY@saucelabs.com/rest/v1/info/status"
 STATUS_URL="https://saucelabs.com/rest/v1/info/status"
 DOWNLOAD_URL="https://saucelabs.com/downloads/$SC_VERSION.tar.gz"
 DOWNLOAD_DIR="sauce_connect"
 TAR_FILE="$DOWNLOAD_DIR/$SC_VERSION.tar.gz"
+TAR_FILE_CHECKSUM="c7ad595a2a42f837fab91de3c3f51b49f2af6b36"
```

Comment:
This will have to be updated with every new version in future. Did you notice problems with corrupted downloads?

Reply:
Updating this shouldn't be a problem when you update the same file with the new version number. There were no issues in the past, but more security can't hurt :)

Reply:
On the contrary, I think this will add friction to the upgrade process. (e.g., need a linux machine, can't use GH web UI)

I'd drop this check.